src/Honeybee_Generator_PV.py

```diff
@@ -57,11 +53,10 @@
 
 ghenv.Component.Name = "Honeybee_Generator_PV"
 ghenv.Component.NickName = 'PVgen'
-ghenv.Component.Message = 'VER 0.0.59\nJAN_26_2016'
-ghenv.Component.IconDisplayMode = ghenv.Component.IconDisplayMode.application
```

Comment:
@antonszilasi sorry for being picky! but you're missing the new line for setting icon default to application.